App.tsx

```diff
@@ -16,6 +16,30 @@ import * as WebBrowser from "expo-web-browser"
 import * as Linking from "expo-linking"
 // import HomeScreen from "./screens/Home/HomeScreen"
 
+import AsyncStorage from "@react-native-async-storage/async-storage"
+
+const GRAPH_STORAGE_KEY = "graph" // Key used by AsyncStorage to store and retrieve the graph file
+const GRAPH_EXISTENCE_FLAG_KEY = "graph_exists" // Key used by AsyncStorage to store a flag indicating whether the graph file exists
+
+// Function to destroy the graph file if it exists
+const destroyGraphFileIfExists = async () => {
+  try {
+    // Check if the graph file exists
+    const graphExists = await AsyncStorage.getItem(GRAPH_EXISTENCE_FLAG_KEY)
+    if (graphExists === "true") {
+      // If the graph file exists, delete it
+      await AsyncStorage.removeItem(GRAPH_STORAGE_KEY)
+      // Reset the flag to indicate that the graph file no longer exists
+      await AsyncStorage.setItem(GRAPH_EXISTENCE_FLAG_KEY, "false")
+    }
+  } catch (error) {
+    console.error("Error destroying graph file:", error)
+  }
+}
+
+// Call the function to destroy the graph file when the app launches
+destroyGraphFileIfExists()
```

Comment:
shouldn't this function be in another file? Ideally we do not want function implementations in App.tsx